src/neo/SmartContract/Native/NeoToken.cs

```diff
@@ -319,6 +381,7 @@ private async ContractTask<bool> Vote(ApplicationEngine engine, UInt160 account,
             if (!engine.CheckWitnessInternal(account)) return false;
             NeoAccountState state_account = engine.Snapshot.GetAndChange(CreateStorageKey(Prefix_Account).Add(account))?.GetInteroperable<NeoAccountState>();
             if (state_account is null) return false;
+            if (state_account.Balance == 0) return false;
```

Comment:
We don't need this line. Because if the balance is 0, the account is removed. And it returns `false` at L383.